src/axom/mint/tests/CMakeLists.txt

```diff
@@ -40,6 +39,13 @@ set( mint_tests
      mint_util_write_vtk.cpp
    )
 
+if(NOT (("${CMAKE_CXX_COMPILER_VERSION}" VERSION_EQUAL "9.0.0") AND
+   ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")))
+    # This test causes a compiler seg fault in clang@9.0.0,
+    # codes are still using mint with that compiler though
+    list(APPEND mint_tests mint_mesh_cell_types.cpp)
+endif()
```

Comment:
This works for me. I e-mail John G. about this and waiting to hear back. I'll look into other possible workarounds for this in the near future.